src/app/components/wrapper-components/tutorial-game-wrapper/tutorial-game-wrapper.games.component.spec.ts

```diff
@@ -116,6 +119,7 @@ describe('TutorialGameWrapperComponent (games)', () => {
             const pylosTutorial: TutorialStep[] = new PylosTutorial().tutorial;
             const saharaTutorial: TutorialStep[] = new SaharaTutorial().tutorial;
             const sixTutorial: TutorialStep[] = new SixTutorial().tutorial;
+            const squarzTutorial: TutorialStep[] = new SquarzTutorial().tutorial;
```

Comment:
quid de ticketter le fait que les stepExpectations elles devraient être définies dans un XXXTutorialStepExpectations ? Ça serait bien que la majorité du code relatif aux jeux soit dans les jeux !

Reply:
Ouais j'avoue, tant que la matrice de code est en commun et que dans les XXXTutorialTest contient que la liste de step à tester !
ça diminuerais au passage l'écart entre 'it(' et le nombre de test concrêts :p
Proposeur encodeur ! écrit le petit tickette lo !